DotNetHowToMultipleDataSources/Program.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Azure.Search;
+using Microsoft.Azure.Search.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Rest.Azure;
+
+namespace AzureSearch.SDKHowTo
+{
+    /// <summary>
+    /// Demo of Azure Search indexer for Azure SQL
+    /// </summary>
+    public sealed class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            IConfigurationBuilder builder = new ConfigurationBuilder().AddJsonFile("appsettings.json");
+            IConfigurationRoot configuration = builder.Build();
+
+            if (configuration["SearchServiceName"] == "Put your search service name here")
+            {
+                Console.Error.WriteLine("Specify SearchServiceName in appsettings.json");
+                Environment.Exit(-1);
+            }
+
+            if (configuration["SearchServiceAdminApiKey"] == "Put your primary or secondary API key here")
+            {
+                Console.Error.WriteLine("Specify SearchServiceAdminApiKey in appsettings.json");
+                Environment.Exit(-1);
+            }
+
+            if (configuration["AzureSQLConnectionString"] == "Put your Azure SQL database connection string here")
+            {
+                Console.Error.WriteLine("Specify AzureSQLConnectionString in appsettings.json");
+                Environment.Exit(-1);
+            }
+
+            if (configuration["CosmosDBConnectionString"] == "Put your Cosmos DB connection string here")
+            {
+                Console.Error.WriteLine("Specify CosmosDBConnectionString in appsettings.json");
+                Environment.Exit(-1);
+            }
+
+            SearchServiceClient searchService = new SearchServiceClient(
+                searchServiceName: configuration["SearchServiceName"],
+                credentials: new SearchCredentials(configuration["SearchServiceAdminApiKey"]));
+
+            Console.WriteLine("Creating index...");
+            Index index = new Index(
+                name: "hotels",
+                fields: FieldBuilder.BuildForType<Hotel>());
+            // If we have run the sample before, this index will be populated
+            // We can clear the index by deleting it if it exists and creating
+            // it again
+            bool exists = await searchService.Indexes.ExistsAsync(index.Name);
```

Comment:
@mgottein There's probably a safer way of doing this using `AccessCondition`. This is a quick-and-dirty way of doing this for a sample, but I wonder if we'd be better off showing best practices more often, even when they're not necessary (e.g. -- in this case it's unlikely anyone will be concurrently manipulating the index, so using access conditions/ETags would be overkill).

Your call.

Reply:
I would rather not overcomplicate this sample and leave everything as is. You can create a separate "how to use access conditions" sample, I think there is usefullness in doing it in at least 1 sample

Reply:
Sounds reasonable. We do have a sample for ETags/access conditions here, so we're covered: https://github.com/Azure-Samples/search-dotnet-getting-started/tree/master/DotNetETagsExplainer